### PR TITLE
minor bugs fix

### PR DESCRIPTION
**Pull request by ariane-blk**

### CODE REVIEW

File: amypet/align.py [hunk 1]
```diff
@@ -444,7 +444,7 @@ def align_suvr(
             fnii_aligned.append(frsmpl)
 
             niiim[ifrm, ...] = nimpa.getnii(frsmpl)
-
+        niiim[np.isnan(niiim)] = 0
```
Comment (casperdcl) [inline amypet/align.py:447]: Odd indentation?
```suggestion
niiim[np.isnan(niiim)] = 0
```

**Review by casperdcl — CHANGES_REQUESTED** [review]
Thanks!

File: amypet/align.py (hunk 1 above)
Comment (ariane-blk) [reply amypet/align.py:447]: It should be the same indentation as line 449, is it not the case?

Comment (casperdcl) [reply amypet/align.py:447]: Oops, nvm. I'm trying thr GitHyb mobile app which seems to hide indentation weirdly

**Pull request merged**